Samples/FsTests/ManyUnitTest.fs

```diff
@@ -0,0 +1,25 @@
+module ManyUnitTests
+
+open NUnit.Framework
+open FsUnit
+open System.IO
+open ParsecClone.BinaryCombinator
+open ParsecClone.CombinatorBase
+
+let (bp: BinParser<unit>) = new BinParser<_> (id)
+
+let pByteZero = bp.matchBytes [| 00uy |] |>> (fun _ -> 00uy)
+let pByteOne = bp.matchBytes [| 01uy |] |>> (fun _ -> 01uy)
+
+let parser = parse {
```

Comment:
Instead of having a new file for just this test, can we move this test into https://github.com/devshorts/ParsecClone/blob/master/Samples/FsTests/BinaryUnitTests.fs?  We can define the parser and helper functions inside the test like the other test examples.